x-pack/plugins/apm/scripts/test/api.js

```diff
@@ -73,4 +77,25 @@ const cmd = `node ${inspectArg} ../../../../scripts/${ftrScript} ${grepArg} --co
 
 console.log(`Running ${cmd}`);
 
-childProcess.execSync(cmd, { cwd: path.join(__dirname), stdio: 'inherit' });
+function runTests() {
+  childProcess.execSync(cmd, { cwd: path.join(__dirname), stdio: 'inherit' });
+}
+
+if (argv.times) {
+  const runCounter = { succeeded: 0, failed: 0, remaining: argv.times };
+  times(argv.times, () => {
+    try {
+      runTests();
+      runCounter.succeeded++;
+    } catch (e) {
+      runCounter.failed++;
+    }
+    runCounter.remaining--;
+
+    if (argv.times > 1) {
```

Comment:
can you make sure that you exit with the appropriate status code here (if there is at least one failure)? I know that we don't use it on CI but I don't want to fall into the same trap again.